_pages/terminology-server.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Terminology Server
+permalink: /terminology-server/
+---
+
+TIMS hosts a FHIR server that provides a unique set of ontologies, terminologies, with unique features such as text 
+search and  SSSOM-formatted mappings.
+
+- http://fhir.terminology-services.com
+- http://20.3.198.176:8080/ (mirror)
+
+### Notable features
```

Comment:
@ShahimEssaid I could use your help verifying that what I have in this section looks good / is correct.